.github/dependabot.yml

```diff
@@ -4,14 +4,32 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    ignore:
+    # Ignore controller-runtime as its upgraded manually.
+    - dependency-name: "sigs.k8s.io/controller-runtime"
+    # Ignore cluster-api as its upgraded manually.
+    - dependency-name: "sigs.k8s.io/cluster-api/*"
+    # Ignore k8s and its transitives modules as they are upgraded manually
+    # together with controller-runtime.
+    - dependency-name: "k8s.io/*"
+    - dependency-name: "github.com/onsi/*"
+    labels:
+      - "ok-to-test"
 
   # Enable version updates for Go tools
   - package-ecosystem: "gomod"
     directory: "/hack/tools"
     schedule:
       interval: "weekly"
-
-  - package-ecosystem: "docker"
-    directory: "/"
-    schedule:
-      interval: "weekly"
```

Comment:
why is this removed?

Reply:
This takes care of updating only `golang` in Dockerfile which anyways we update manually, so seems like irrelevant.

Reply:
yes, that's the intention of this code to update the golang version in the dockerfile but seems like it is not working as expected and the reason why we are doing it manually. Lets keep this line as is for now and explore later why it is not updating the version(of course only the patched version)

Reply:
Done.